src/OmniSharp.Roslyn.CSharp/Services/Completion/CompletionService.cs

```diff
@@ -150,7 +151,7 @@ public async Task<CompletionResponse> Handle(CompletionRequest request)
             }
 
             var triggerCharactersBuilder = ImmutableArray.CreateBuilder<char>(completions.Rules.DefaultCommitCharacters.Length);
-            var completionsBuilder = ImmutableArray.CreateBuilder<CompletionItem>(completions.Items.Length);
+            var completionsBuilder = new List<CompletionItem>();
```

Comment:
Consider leaving this as a builder, and instead calling `ToImmutable` on the builder below instead of `MoveToImmutable()` as the builder is no longer guaranteed to be correctly sized.

Reply:
yes thanks it was `ToImmutable()`, just got messed up on the bad merge